apps/api/src/types/index.ts

```diff
@@ -1080,7 +1080,7 @@ export const AdminCreateAgentSchema = z.object({
     walletAddress: z.string().optional(),
     description: z.string().optional(),
     imageUrl: z.string().optional(),
-    metadata: z.nullish(AgentMetadataSchema),
+    metadata: AgentMetadataSchema.optional(),
```

Comment:
this is technically a minor behavior change in the API, but the AI tells me there are no callers in the frontend that explicitly pass null here, and making this optional instead of nullish makes it much more consistent with the rest of the API and the internal types and avoids some extra null->undefined type conversion that is necessary otherwise.